tests/introspection/iOS/iOSApiSelectorTest.cs

```diff
@@ -164,6 +164,16 @@ protected override bool CheckResponse (bool value, Type actualType, MethodBase m
 					break;
 				}
 				break;
+			case "SKNode":
+				switch (name) {
+				case "didUpdateFocusInContext:withAnimationCoordinator:":
+				case "setNeedsFocusUpdate":
+				case "shouldUpdateFocusInContext:":
+				case "updateFocusIfNeeded":
+				case "preferredFocusedView":
+					return true;
```

Comment:
add an Xcode check, not just a return, so testing on older platform is not affected